src/main/kotlin/string/addition/caculator/domain/Operand.kt

```diff
@@ -0,0 +1,21 @@
+package string.addition.caculator.domain
+
+data class Operand(private val inputStr: String) {
+    var number: Int
+        private set
+
+    init {
+        require(inputStr.toIntOrNull() != null) {
+            throw RuntimeException("An operand from the input string should be a Number")
+        }
+        number = inputStr.toInt()
```

Comment:
```suggestion
        number = inputStr.toIntOrNull() ?:  RuntimeException("An operand from the input string should be a Number")
```

이렇게도 표현할 수 있을 거 같아요. 앞으로 자주 보거나 사용하게 될 코틀린의 예외처리가 포함된 값 할당 방식입니다 😄